resources/base.css

```diff
@@ -19,7 +19,7 @@ a.link-small {
 
 ul {
   list-style: none;
-  padding-left: 26px;
+  columns: 2;
```

Comment:
Why this change?

Reply:
I rearranged the index page and this gives much nicer layout.